app/views/docs/self-hosting.phtml

```diff
@@ -1,4 +1,4 @@
-<p>Appwrite is a self-hosted backend server, packaged as a set of Docker containers. You can install and run Appwrite on any operating system that can run a <a href="https://www.docker.com/products/docker-desktop" target="_blank" rel="noopener">Docker CLI</a>. You can use Appwrite on your local machine or a cloud provider of your choice.</p>
+<p>Appwrite can be easily self-hosted on your local machine or a server. You can install and run Appwrite on any operating system that can run a <a href="https://www.docker.com/products/docker-desktop" target="_blank" rel="noopener">Docker CLI</a>. Self-hosted Appwrite instances can be configured flexibly with access to the same features found on Appwrite Cloud.</p>
```

Comment:
let's rephrase it and say something along the lines of "Appwrite was designed from the ground up with self-hosting in mind"

Reply:
Why don't we just say Appwrite was designed from the ground up with self-hosting in mind, I like it XD

Reply:
Sounds good 👌